source/viz.md

```diff
@@ -1104,10 +1189,16 @@ Here, we have a data frame of Earth's landmasses,
 and are trying to compare their sizes.
 The right type of visualization to answer this question is a bar plot.
 In a bar plot, the height of the bar represents the value of a summary statistic
-(usually a size, count, proportion or percentage).
+(usually a size, count, sum, proportion or percentage).
 They are particularly useful for comparing summary statistics between different
 groups of a categorical variable.
 
+> **Note:** Although bar charts are often used to display mean and median values,
+> they are generally a poor choice for these types of visualizations and should be avoided.
```

Comment:
I think this nuance is too much for this relatively early chapter and will confuse a lot of our students. I suggest removing it. 

It's a good point though. If we expanded our chapter with boxplots, I think that's where you'd make this argument w/o losing people. perhaps open an issue to add a section on boxplots, copy/paste this text into that issue so we can save it for later, and delete the note box here.

Reply:
Sorry I missed this comment initially. I think this is more than a nuance and a common mistake I see students making, so I would be in favor of adding it to the R and Python books. I think it could fit nicely as a motivation for the histogram section, which could begin by comparing the average and then showing the actual histograms of all observations instead. What do you think of that?

Reply:
I'm still not entirely convinced...

Idea: can we open a separate PR (or an issue) to discuss this change? I don't want to block this PR on that

Reply:
Opened https://github.com/UBC-DSCI/introduction-to-datascience-python/issues/178